### PR TITLE
ZEUS-2053: Android: Tooltip text hidden in Dark Theme mode

### PR DESCRIPTION
**Pull request by kaloudis**
# Description

Relates to issue: [**ZEUS-2053**](https://github.com/ZeusLN/zeus/issues/2053)

This pull request is categorized as a:

- [ ] New feature
- [x] Bug fix
- [ ] Code refactor
- [ ] Configuration change
- [ ] Locales update
- [ ] Quality assurance
- [ ] Other

## Checklist
- [ ] I’ve run `yarn run tsc` and made sure my code compiles correctly
- [x] I’ve run `yarn run lint` and made sure my code didn’t contain any problematic patterns
- [x] I’ve run `yarn run prettier` and made sure my code is formatted correctly
- [x] I’ve run `yarn run test` and made sure all of the tests pass

## Testing

If you modified or added a utility file, did you add new unit tests?

- [ ] No, I’m a fool
- [ ] Yes
- [x] N/A

I have tested this PR on the following platforms (please specify OS version and phone model/VM):

- [x] Android
- [ ] iOS

I have tested this PR with the following types of nodes (please specify node version and API version where appropriate):

- [x] Embedded LND
- [ ] LND (REST)
- [ ] LND (Lightning Node Connect)
- [ ] Core Lightning (c-lightning-REST)
- [ ] LndHub
- [ ] [DEPRECATED] Core Lightning (Spark)
- [ ] [DEPRECATED] Eclair

### Locales
- [ ] I’ve added new locale text that requires translations
- [ ] I’m aware that new translations should be made on the ZEUS [Transfix page](https://app.transifex.com/ZeusLN/zeus/) and not directly to this repo

### Third Party Dependencies and Packages

- [ ] Contributors will need to run `yarn` after this PR is merged in
- [ ] 3rd party dependencies have been modified:
    * verify that `package.json` and `yarn.lock` have been properly updated
    * verify that dependencies are installed for both iOS and Android platforms

### Other:

- [ ] Changes were made that require an update to the README
- [ ] Changes were made that require an update to onboarding

### CODE REVIEW

File: components/Modals/InfoModal.tsx [hunk 1]
```diff
@@ -63,6 +63,7 @@ export default class InfoModal extends React.Component<InfoModalProps, {}> {
                             <Text
                                 style={{
                                     fontFamily: 'PPNeueMontreal-Book',
+                                    color: 'black',
```
Comment (shubhamkmr04) [inline components/Modals/InfoModal.tsx:66]: can we use `themeColor('background')` here instead ?

Comment (kaloudis) [reply components/Modals/InfoModal.tsx:66]: Potentially, but we'd also have to change the the value set for `white` as the modal background here too

**Comment by kaloudis** [comment]
Redone with theme colors:

simulator_screenshot_A3EF3382-FC37-4684-BF4C-C44C6DEA5E22

simulator_screenshot_053B46C0-5258-47B9-9C00-5B18AA70DCDA

simulator_screenshot_897E404C-64B4-482B-AD42-9CF26FE71EBA

**Review by shubhamkmr04 — COMMENTED** [review]
tACK

**Pull request merged**